lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
+                        return that.$request_err_grade.text(rescoreUnsupportedMessage);
```

Comment:
How about a "this string has strings attached" warning comment at https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor_task/api_helper.py#L261 for the sake of posterity.